tests/apps/demultiplex/conftest.py

```diff
@@ -347,3 +347,17 @@ def fixture_novaseq6000_flow_cell_sample_before_adapt_indexes() -> (
         SampleName="814206",
         Project="814206",
     )
+
+
+@pytest.fixture(name="novaseq6000_sample_sheet_path")
+def fixture_novaseq6000_sample_sheet_path() -> Path:
+    """Return the path to a NovaSeq S2 sample sheet."""
```

Comment:
If this is the Flow cell mode, it was deprecated some time ago so I think it would be ok to remove that and replace it for NovaSeq6000

Reply:
There is also a difference between bcl2fastq and BCLConvert sample sheets. So maybe good to mention that in the docstring too.

Reply:
If you could advise me on the what version the sample sheet is we can add it, I honestly do not know the difference.

I think for the functionality of the function it does not matter since the we check a field that is common in all sample sheets judging from the flowcellsample models. We do not test for the parsing itself, but whether an already parsed attribute follows a structure.